CHANGELOG.md

```diff
@@ -44,6 +44,11 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ### Linter
 
+#### New features
+
+- Add [nursery/noIrregularWhitespace](https://biomejs.dev/linter/rules/no-irregular-whitespace). Contributed by @michellocana
+
+
```

Comment:
I think there are already `New Features section`, below Enhancements.

Reply:
There is also a `New rules` section. We should move `New rules` and `New features` before `Enhancements`. Also, we should merge `New rules` and `New features` into `New features` - this is what we usually do.